x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="A GUI for f.lux"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="https://github.com/xflux-gui/fluxgui/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="dev-libs/libappindicator:2[python,${PYTHON_USEDEP}]
+	dev-python/gconf-python[${PYTHON_USEDEP}]
+	dev-python/pexpect[${PYTHON_USEDEP}]
+	dev-python/pygtk[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	gnome-base/libglade[${PYTHON_USEDEP}]
```

Comment:
You went one package too far. Here PYTHON_TARGETS are only used for USE=tools.

Reply:
(but I'll fix that, no point in you remaking the PR for it)

Reply:
Whoops, sorry, next time, I will look through the ebuild deps, when looking through python deps.